zazu/repo/commands.py

```diff
@@ -12,7 +14,9 @@
 @click.pass_context
 def repo(ctx):
     """Manage repository"""
-    ctx.obj.check_repo()
+    if not ctx.invoked_subcommand == 'clone':
```

Comment:
Probably better to add this check to all commands that need it rather than exempt just one.